src/cdomain/value/cdomains/int/intervalSetDomain.ml

```diff
@@ -236,6 +236,28 @@ struct
 
   let of_interval ?(suppress_ovwarn=false) ik (x,y) =  norm_interval  ~suppress_ovwarn ~cast:false ik (x,y)
 
+  let of_bitfield ik x = 
+    let min ik (z,o) = 
+      let signBit = Ints_t.shift_left Ints_t.one ((Size.bit ik) - 1) in 
+      let signMask = Ints_t.lognot (Ints_t.of_bigint (snd (Size.range ik))) in
```

Comment:
`signBit` and `signMask` only seem to depend on `ik'. Can one extract them from `min` and `max`, so they don't have to be defined twice?